File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/app/d/CiAppTreatSheetOrDO.java
package iih.ci.ord.app.d;

import xap.sys.appfw.orm.desc.DescManager;
import xap.mw.coreitf.c.GlobalConst;
import xap.mw.coreitf.i.IDODesc;
import xap.mw.coreitf.d.*;
import xap.mw.core.data.*;
import iih.ci.ord.app.d.desc.CiAppTreatSheetOrDODesc;
import java.math.BigDecimal;

/**
 * 诊疗申请单对应医嘱 DO数据 
 * 
 */
public class CiAppTreatSheetOrDO extends BaseDO {
	private static final long serialVersionUID = 1L;

	public static final String ID_CIAPPTREATSHEETOR= "Id_ciapptreatsheetor";
	public static final String ID_CIAPPTREATSHEET= "Id_ciapptreatsheet";
	public static final String ID_OR= "Id_or";
	public static final String AMT_OR= "Amt_or";
	public static final String DS = "Ds";
	public static final String SV = "Sv";
	
	public String getId_ciapptreatsheetor() {
		return ((String) getAttrVal("Id_ciapptreatsheetor"));
	}	
	public void setId_ciapptreatsheetor(String Id_ciapptreatsheetor) {
		setAttrVal("Id_ciapptreatsheetor", Id_ciapptreatsheetor);
	}
	public String getId_ciapptreatsheet() {
		return ((String) getAttrVal("Id_ciapptreatsheet"));
	}	
	public void setId_ciapptreatsheet(String Id_ciapptreatsheet) {
		setAttrVal("Id_ciapptreatsheet", Id_ciapptreatsheet);
	}
	public String getId_or() {
		return ((String) getAttrVal("Id_or"));
	}	
	public void setId_or(String Id_or) {
		setAttrVal("Id_or", Id_or);
	}
	public FDouble getAmt_or() {
		return ((FDouble) getAttrVal("Amt_or"));
	}	
	public void setAmt_or(FDouble Amt_or) {
		setAttrVal("Amt_or", Amt_or);
	}

	public Integer getDs() {
		return ((Integer) getAttrVal(GlobalConst.DELETE_SIGN));
	}
	
	public void setDs(Integer ds) {
		setAttrVal(GlobalConst.DELETE_SIGN, ds);
	}
	
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal(GlobalConst.SYSTEM_VERSION));
	}
	
	public void setSv(FDateTime sv) {
		setAttrVal(GlobalConst.SYSTEM_VERSION, sv);
	}

//	@Override
//	public java.lang.String getParentPKFieldName() {
//		return null;
//	}
  
	@Override
	public String getPKFieldName() {
		return "Id_ciapptreatsheetor";
	}
	
	@Override
	public String getTableName() {	  
		return "ci_app_treat_or";
	}
	
	@Override
	public IDODesc getDODesc() {
		return DescManager.getInstance().getDODesc(CiAppTreatSheetOrDODesc.class);
	}
	
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/app/d/CiAppTreatSheetOrDO.cs

using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.app.d
{
    /// <summary>
    /// CiAppTreatSheetOrDO 的摘要说明。
    /// </summary>
    public class CiAppTreatSheetOrDO : BaseDO {

		public const string TABLE_NAME = "ci_app_treat_or";
		public const string TABLE_ALIAS_NAME = "a1";

        public CiAppTreatSheetOrDO() {
        }
		public string Id_ciapptreatsheetor {
            get { return getAttrVal<string>("Id_ciapptreatsheetor",null); }
            set { setAttrVal<string>("Id_ciapptreatsheetor", value); }
        }
		public string Id_ciapptreatsheet {
            get { return getAttrVal<string>("Id_ciapptreatsheet",null); }
            set { setAttrVal<string>("Id_ciapptreatsheet", value); }
        }
		public string Id_or {
            get { return getAttrVal<string>("Id_or",null); }
            set { setAttrVal<string>("Id_or", value); }
        }
		public FDouble Amt_or {
            get { return getAttrVal<FDouble>("Amt_or",null); }
            set { setAttrVal<FDouble>("Amt_or", value); }
        }
        public int Ds {
            get { return getAttrVal<int>("Ds",0);}
            set { setAttrVal<int>("Ds", value); }
        }

        public DateTime? Sv        {
            get { return getAttrVal<FDateTime>("Sv",null); }
            set { setAttrVal<FDateTime>("Sv", value); }
        }
        
        /// <summary>
        /// 返回表名
        /// </summary>
        /// <returns></returns>
        public override string getTableName()
        {
            return "ci_app_treat_or";
        }
        
        /// <summary>
        /// 返回表别名
        /// </summary>
        /// <returns></returns>
        public override string getAliasTableName()
        {
            return "a1";
        }
        
        /// <summary>
        /// 返回主键字段名
        /// </summary>
        /// <returns></returns>
        public override string getPKFieldName()
        {
            return "Id_ciapptreatsheetor";
        }

        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.app.d.CiAppTreatSheetOrDO";
        }

        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{"Ds","Sv",  "Id_ciapptreatsheetor", "Id_ciapptreatsheet", "Id_or", "Amt_or"};
        }
        
    }
}
